src/test/java/tw/edu/ntu/lowerbound10hours/jlaskhibernate/DatabaseOperationTest.java

```diff
@@ -0,0 +1,34 @@
+package tw.edu.ntu.lowerbound10hours.jlaskhibernate;
+
+import static org.testng.Assert.assertEquals;
+
+import java.util.List;
+import org.hibernate.Criteria;
+import org.testng.annotations.Test;
+import tw.edu.ntu.lowerbound10hours.jlask.Jlask;
+
+public class DatabaseOperationTest {
+
+  private Hibernate db = new Hibernate();
+
+  public DatabaseOperationTest() {
+    db.drop_all();
+    db.init_app(new Jlask());
+  }
+
+  @Test
+  public void testInsert() {
```

Comment:
Is this an insert operation? It looks like a table creation

Reply:
It is the insert operation right. If the table doesn't exist in the DB, it would be created automatically when inserting data.